### ISSUE TITLE
[Feature Request] Jump to character count

### ISSUE THREAD

**Issue by Lyroxide** [ttu-ttu/ebook-reader#350]
Some books have the entire text as one single chapter. Would be nice to be able to jump to a certain character count / percentage of the book

**Comment by sdaqo** [comment]
I agree, there is a lack of book navigation options. I think an interactive slider at the bottom (hideable like the top bar) indicating progress would be awesome.
